corehq/apps/integration/payments/tests/test_views.py

```diff
@@ -69,6 +78,7 @@ def test_ff_not_enabled(self):
         assert response.status_code == 404
 
     @flag_enabled('MTN_MOBILE_WORKER_VERIFICATION')
+    @patch.object(BatchNumberFilter, 'options', [])
```

Comment:
nit: I assume this is returning the options to select from? Should there be 1 or 2 options to also check that there are no issues rendering them?